app/src/main/java/com/infomaniak/mail/ui/newMessage/NewMessageViewModel.kt

```diff
@@ -110,8 +114,16 @@ class NewMessageViewModel @Inject constructor(
     val uiQuoteLiveData = MutableLiveData<String?>()
     //endregion
 
-    var lastOnStopSubjectValue = ""
-    var lastOnStopBodyValue = ""
+    val editorBodyLoader = SingleLiveEvent<BodyContentPayload>()
```

Comment:
A loader that contains data ?

Maybe the `loader` part of the name is a little confusing.

Reply:
I tried adding something to the name so when reading where this value is used, we understand that this live data is not meant for keeping the content of the editor up to date or anything but only to initialize it.

Do you think that something more along the lines of `editorBodyInitializer` be easier to understand?

Reply:
This is the only way for the draft's content to actually be loaded

Reply:
Fabian also wanted this, so I already did it